packages/@lwc/shared/src/language.ts

```diff
@@ -171,13 +172,14 @@ export function toString(obj: any): string {
         }
         return obj.toString();
     } else if (typeof obj === 'object') {
+        // Oops! This catches null and returns "[object Null]"
         return OtS.call(obj);
     } else {
-        return obj + '';
+        return String(obj);
```

Comment:
Side effect of ditching `any`, since `+` can't be used with `unknown`. This case should just be `undefined`, though, so this is an inconsequential change.